app/src/main/java/com/pajato/android/gamechat/chat/ChatFragment.java

```diff
@@ -441,12 +441,12 @@ public boolean onMenuItemSelected(MenuItem menuItem) {
                     // Start a new Tic-Tac-Toe game
                     case R.id.fab_ttt:
                         if(viewPager != null) { viewPager.setCurrentItem(PaneManager.GAME_INDEX); }
-                        GameManager.instance.sendNewGame(GameManager.TTT_INDEX, getActivity());
+                        GameManager.instance.sendNewGame(GameManager.TTT_L_INDEX, getActivity());
```

Comment:
TTT_L_INDEX is a tad cryptic.  Expand on the L part a bit to make it more obvious what's going on.